app/javascript/spec/ansible-repository-form/ansible-repository-form.spec.js

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import toJson from 'enzyme-to-json';
+import fetchMock from 'fetch-mock';
+import { shallow } from 'enzyme';
+import AnsibleRepositoryForm from '../../components/ansible-repository-form';
+
+require('../helpers/set_fixtures_helper.js');
+require('../helpers/old_js_file_require_helper.js');
+require('../helpers/miqSparkle.js');
+require('../helpers/miqAjaxButton.js');
+
+jest.mock('../../helpers/miq-redirect-back', () => jest.fn());
+
+describe('Ansible Repository form component', () => {
+  let submitSpyMiqSparkleOn;
+  let submitSpyMiqSparkleOff;
+  let spyMiqAjaxButton;
```

Comment:
@GilbertCherrie I don't think we need `spyMiqAjaxButton` in this file as we are not using `MiqAjaxButton` in react file.